deployment/packer/template.js

```diff
@@ -48,6 +48,7 @@
                 "Service": "Tiler",
                 "Environment": "{{user `stack_type`}}"
             },
+            "subnet_id": "subnet-3781f46e",
```

Comment:
Right. Didn't realize it was pure json since the file has a normal js extension.